src/pybind/mgr/orchestrator/module.py

```diff
@@ -380,6 +380,13 @@ def _get_hosts(self, format: Format = Format.plain) -> HandleCommandResult:
             output = table.get_string()
         return HandleCommandResult(stdout=output)
 
+    @_cli_read_command('orch host gather facts')
+    def _get_hosts_gather_facts(self, hostname) -> HandleCommandResult:
+        """List hosts metadata"""
+        completion = self.get_gather_facts(hostname)
+        raise_if_exception(completion)
+        return HandleCommandResult(stdout=completion.result_str())
```

Comment:
The problem is, if you expose it on the CLI, you'll proablay want to clean the facts up. I'd avoid it for now.

Reply:
Removed from here. Thanks